src/cryptonote_core/service_node_list.cpp

```diff
@@ -1215,6 +1215,18 @@ namespace service_nodes
         if (nettype == cryptonote::TESTNET && state.height < 85357)
           total_nodes = active_snode_list.size() + decomm_snode_list.size();
 
+
+        // TODO(loki): We can remove after switching to V13 since we delete all V12 and below checkpoints where we introduced this kind of quorum
+        if (hf_version >= cryptonote::network_version_13_enforce_checkpoints && total_nodes < CHECKPOINT_QUORUM_SIZE)
+        {
+          if ((nettype == cryptonote::TESTNET && state.height >= 129120))
```

Comment:
I *think* we might get away fine without this soft fork condition being necessary here -- we've had minimum quorum since the HF height AFAIK.

Reply:
We will need to forgo softforking and rollback testnet because the reward penalty fix breaks testnet since its already underpaid before on HF13, so I can remove this check altogether.